.github/workflows/continuous-integration.yml

```diff
@@ -84,17 +78,8 @@ jobs:
           extensions: mbstring
           php-version: ${{ matrix.php-version }}
 
-      - name: "Install lowest dependencies with composer"
-        if: matrix.dependencies == 'lowest'
-        run: composer update --prefer-lowest --no-interaction --no-progress --no-suggest
-
-      - name: "Install locked dependencies with composer"
-        if: matrix.dependencies == 'locked'
-        run: composer install --no-interaction --no-progress --no-suggest
-
-      - name: "Install highest dependencies with composer"
-        if: matrix.dependencies == 'highest'
-        run: composer update --no-interaction --no-progress --no-suggest
+      - name: "Install dependencies with composer"
```

Comment:
Why did you change this part?

Reply:
`matrix.dependencies == 'locked'` will not work if `composer.lock` is not committed.

Thus, only `lowest` and `highest` will actually work. Besides, all dependencies are use stable version, `highest` is equal to `--prefer-stable`.

Thus, I simplified this section.